indexing-service/src/main/java/org/apache/druid/indexing/input/DruidInputSource.java

```diff
@@ -179,6 +183,14 @@ public DruidInputSource(
     this.taskConfig = Preconditions.checkNotNull(taskConfig, "null taskConfig");
   }
 
+  @JsonIgnore
+  @Nonnull
+  @Override
+  public Set<String> getTypes()
+  {
+    return ImmutableSet.of();
```

Comment:
Should be `ImmutableSet.of("druid")`, otherwise permissions won't be appropriately checked.

Reply:
Thanks! updated